registers/TUTORIAL.md

```diff
@@ -14,3 +14,45 @@ You can run it with
 
 By default it uses the "register" register (a list of all registers)
 but you can point it at others with the ```--register``` flag.
+
+Step 2
+------
+
+Next, we are going to put the data into a Trillian log. The first step
```

Comment:
nit: I've been trying to capitalise Log and Map just to make it clear(er) to people arriving to this stuff for the first time that it's not just a file of stuff.

Reply:
Hmm. There's guidance somewhere about not capitalising everything we Invent because we think it is Special.

Reply:
If it were Guidance I might've read it :)